_posts/2016-03-02-summer-of-code.md

```diff
@@ -0,0 +1,36 @@
+---
+layout:     post
+title:      Servo and Summer of Code
+date:       2016-03-02 20:30:00
+summary:    Get a chance to work on a substantial project in Servo this summer!
+categories:
+---
+
+
+This summer, [Servo][servo] is participating in both [Google Summer of Code][gsoc] and 
+[Rails Girls Summer of Code][rgsoc].
+
+If you're a student with a passion for open source and want to work on a 3-month mentored Rust
+project in Servo (with a stipend), consider applying!
+
+The Rails Girls Summer of Code project ideas can be found [here][rgsoc-servo], and the Google Summer
+of Code ideas are [here][gsoc-servo]. Applications need not be confined to these ideas; if you have
+something else you want to work on let us know.
+
```

Comment:
Let's add the specific application deadlines for each of the programs in this post, too.